cmd/podman/spec.go

```diff
@@ -356,6 +357,13 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		return nil, err
 	}
 
+	// BLOCK IO
+	blkio, err := config.CreateBlockIO()
```

Comment:
Does this happen no matter what?

Reply:
I added a check after to see if `blkio` is nil or not, and if its not it is added to the spec. Is that better @rhatdan ? Because a bunch of flags are used in the `CreateBlockIO` function, which would be a huge if statement.

Reply:
SGTM